src/platform/macos/system/ffi.rs

```diff
@@ -4,35 +4,8 @@
 
 #![allow(non_upper_case_globals)]
 
-use io_surface::IOSurfaceRef;
-use mach2::kern_return::kern_return_t;
-use std::os::raw::c_void;
-
-pub(crate) const kCVPixelFormatType_32BGRA: i32 = 0x42475241; // 'BGRA'
-
-pub(crate) const kCVReturnSuccess: i32 = 0;
-
 pub(crate) const kIODefaultCache: i32 = 0;
 pub(crate) const kIOWriteCombineCache: i32 = 4;
 pub(crate) const kIOMapCacheShift: i32 = 8;
 pub(crate) const kIOMapDefaultCache: i32 = kIODefaultCache << kIOMapCacheShift;
 pub(crate) const kIOMapWriteCombineCache: i32 = kIOWriteCombineCache << kIOMapCacheShift;
```

Comment:
Provided in `objc2-io-kit`, but felt like too little benefit to take on the dependency.